weed/weed.go

```diff
@@ -49,6 +51,19 @@ func main() {
 	glog.MaxFileCount = 5
 	flag.Usage = usage
 
+	err := sentry.Init(sentry.ClientOptions{
```

Comment:
Let's initialize if the corresponding environment variables are declared?

Reply:
ok. I'll add it in next PR with tweaks to report rates.
Basically sentry sdk will not initialize itself when DSN is empty https://github.com/getsentry/sentry-go/blob/424b37d3354557bab8f4e7eb08c3bb6518bbd489/client.go#L328

But some linters expect to handle returned errors anyway.